### ISSUE TITLE
Discuss Additional Navigation Options

### ISSUE THREAD

**Issue by tropicandid** [forumone/gesso-theme-wordpress#429]
1. Something that has not yet come up on many new builds has been the implementation of a mega menu. One solution that was put forth was this tool: https://wordpress.org/plugins/megamenu/
2. We recently had Robbie working on a more accessibility friendly php constructed menu for sites that cannot leverage template parts/the navigation block for their menus. Let's discuss how / if we could incorporate that into the gesso baseline for WP.

**Comment by tropicandid** [comment]
* Ability to truly lock layouts in place
* Manage pulling content down between sites or making sure content doesn't blow away settings on deploy

**Comment by tropicandid** [comment]
https://learn.wordpress.org/tutorial/the-key-to-locking-blocks/ looks like we might be able to control unlocking
